packages/core-modules/contracts/interfaces/IElectionModule.sol

```diff
@@ -116,4 +122,13 @@ interface IElectionModule {
     function getBallotVotes(bytes32 ballotId) external view returns (uint);
 
     function getBallotCandidates(bytes32 ballotId) external view returns (address[] memory);
+
+    // Resolutions
+    // ~~~~~~~~~~~~~~~~~~
+
+    function isElectionEvaluated() external view returns (bool);
+
+    function getCandidateVotes(address candidate) external view returns (uint);
```

Comment:
nit: getCandidateVotes will only apply to certain counting (identifying winners) strategies. Nothing that hurts right now but to keep in mind for the future

Reply:
Hmm... ok. This will expose whatever counting algorithms store in the candidateVotes mapping, within the storage namespace. If this will not be used in other strategies, then this is not a nit, and we need to review our strategy.

Reply:
I'd say, keep it that way until we implement another strategies. Maybe just overriding those functions on the other implementations is enough